nixos/tests/systemd.nix

```diff
@@ -24,6 +24,10 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     services.journald.extraConfig = "Storage=volatile";
     test-support.displayManager.auto.user = "alice";
 
+    # To ensure systemd has been compiled with the
+    # systemd-journal-gatewayd support.
+    services.journald.enableHttpGateway = true;
```

Comment:
Can we test for some service being enabled and running, and not just enable this option?

Reply:
I added a http request to ensure the journal-gatewayd service is up and running.

Reply:
Thanks!